certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/message/MessageController.java

```diff
@@ -106,4 +118,25 @@ SendMessageResponse sendMessage(
       @PathVariable("messageId") String messageId) {
     return sendMessageService.send(request, messageId);
   }
+
+  @PostMapping("/{messageId}/saveanswer")
```

Comment:
Should we have made a AnswerController instead so the endpoint could be /answer/save instead?

Reply:
Hmmm... I kinda like the idea, but im not sure it makes sense. 

Reason for this beeing that the messageId that is provided is not connected to the answer, but to the message. We then later use that id to retrieve the Message and update the answer.

To me thats a good argument to keep it in the MessageController, since the services handle Messages. Do you agree?